src/ocamlorg_frontend/url.ml

```diff
@@ -1,6 +1,7 @@
 let index = "/"
 let packages = "/packages"
 let packages_search = "/packages/search"
+let packages_autocomplete_f = "/packages/autocomplete"
```

Comment:
Is `_f` required? I don't get what it means, what it's used for.

Reply:
intention is to signify that this is not a full HTML page but only a fragment. Could possibly mark that better :thinking: